locA: [1.3716, 0.254]
locB: [1.0668, 0.254]
locC: [0.7874, 0.254]
locD: [0.4826, 0.254]
locE: [0.1778, 0.254]
locF: [0.254, 0.1778]
locG: [0.254, 0.4318]
locH: [0.254, 0.2032]
